bindata/cloud-network-config-controller/managed/controller.yaml

```diff
@@ -26,6 +29,32 @@ spec:
         type: infra
         openshift.io/component: network
     spec:
+      affinity:
+        nodeAffinity:
+          preferredDuringSchedulingIgnoredDuringExecution:
+            - weight: 50
```

Comment:
I am curious about this weight. Can there be nodes labeled with `hypershift.openshift.io/cluster={{.HostedClusterNamespace}}`  but not labeled with `hypershift.openshift.io/control-plane`?

Reply:
IIURC `hypershift.openshift.io/control-plane` is a general label for all HCP pods that should have a lower priority than the more specific, per cluster `hypershift.openshift.io/cluster`.
In theory, I think it might be possible to only see `hypershift.openshift.io/cluster` but @enxebre might have more info on real world scenarios.